### ISSUE TITLE
[1.12.2] Spider navigation issue with the Epic Siege mod

### ISSUE THREAD

**Issue by Cephelo** [TheCyberBrick/Spiders-2.0#47]
Crashes worlds very quickly when better spiders begin to spawn.  I cannot enter any world where the crash has occurred.  If you need more information, contact me on discord at Cephelo#4566
https://pastebin.com/UY3d6cSC

**Comment by Cephelo** [comment]
Funwayguy, the developer for Epic Siege, believes the crash originated from "the error originating in their custom navigation code.", if that helps at all.

**Comment by TheCyberBrick** [comment]
Yes, he's right. I've seen this before, but not yet sure what exactly is causing it.
